src/mmw/sass/pages/_data-catalog.scss

```diff
@@ -46,9 +69,9 @@
         }
         .data-catalog-search-icon {
             color: $ui-grey;
-            position: relative;
```

Comment:
I'm not sure what this means, Matt. I'm going to need some developer support to get the pagination fixed in the correct way. Previously, the pagination didn't move, which was good, but there were other issues with the sidebar layout. I fixed the other issues but I need help to get the pagination working right.